evernode-governor-hook/src/governor.c

```diff
@@ -851,6 +829,39 @@ int64_t hook(uint32_t reserved)
                 PERMIT_M("Evernode: Piloted mode candidate status changed.", vote_status);
             }
         }
+
+        if (redirect_op_type == OP_CHANGE_CONFIGURATION)
+        {
+            SET_UINT_STATE_VALUE(DEF_MOMENT_SIZE, CONF_MOMENT_SIZE, "Evernode: Could not initialize state for moment size.");
+            SET_UINT_STATE_VALUE(DEF_MINT_LIMIT, CONF_MINT_LIMIT, "Evernode: Could not initialize state for mint limit.");
+            SET_UINT_STATE_VALUE(DEF_FIXED_REG_FEE, CONF_FIXED_REG_FEE, "Evernode: Could not initialize state for fixed reg fee.");
+            
+            // <epoch_count(uint8_t)><first_epoch_reward_quota(uint32_t)><epoch_reward_amount(uint32_t)><reward_start_moment(uint32_t)><accumulated_reward_frequency(uint16_t)>
+            uint8_t reward_configuration[REWARD_CONFIGURATION_VAL_SIZE];
```

Comment:
Use global `reward_configuration` initialized on line 272. Then add another condition `op_type == OP_HOOK_UPDATE` in line 273 if condition. So it would read the existing configurations. So it would take the exisiting state and only the offsets which need to be updated will be populated here.